File: packages/pi-structured-return/benchmarks/test-runners/java/src/test/java/MathTest.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

class MathTest {

    @Test
    void addsTwoNumbersCorrectly() {
        assertEquals(2, 1 + 1);
    }

    @Test
    void multipliesTwoNumbersCorrectly() {
        assertEquals(99, 3 * 4);
    }

    @Test
    void doesNotDivideByZero() {
        int result = 1 / 0;
        assertEquals(5, result);
    }
}

File: packages/pi-structured-return/benchmarks/test-runners/dotnet/MathTest.cs
namespace Benchmark;

public class MathTest
{
    [Fact]
    public void AddsTwoNumbersCorrectly()
    {
        Assert.Equal(2, 1 + 1);
    }

    [Fact]
    public void MultipliesTwoNumbersCorrectly()
    {
        Assert.Equal(99, 3 * 4);
    }

    [Fact]
    public void DoesNotThrowOnNullAccess()
    {
        string? s = null;
        int result = s!.Length;
        Assert.Equal(5, result);
    }
}
